CHANGES.rst

```diff
@@ -268,9 +212,42 @@ Contributor-facing changes
 
 
 
+Removals and backward incompatible breaking changes
+---------------------------------------------------
+
+- The shutdown logic in 3.9 waited on all tasks, which caused issues with some libraries.
+  In 3.10 we've changed this logic to only wait on request handlers. This means that it's
+  important for developers to correctly handle the lifecycle of background tasks using a
+  library such as ``aiojobs``. If an application is using ``handler_cancellation=True`` then
+  it is also a good idea to ensure that any :func:`asyncio.shield` calls are replaced with
+  :func:`aiojobs.aiohttp.shield`.
+
+  Please read the updated documentation on these points:
+  https://docs.aiohttp.org/en/stable/web_advanced.html#graceful-shutdown
+  https://docs.aiohttp.org/en/stable/web_advanced.html#web-handler-cancellation
```

Comment:
@bdraco FYI subsequent lines are merged into continuous paragraphs in RST. This is basically rendered as a single line: https://aiohttp--8550.org.readthedocs.build/en/8550/changes.html#removals-and-backward-incompatible-breaking-changes.
Did you expect this to be a list?

Perhaps, it's worth adding a changelog render preview step to the release checklist?

Reply:
I made the links a list

Reply:
I'm not sure how to fix this one as it doesn't seem RST supports list inside of a list.  I've tried line blocks `|`, `|br|` and back slashes.

Reply:
Nevermind, it does look like the backslashes worked.